### PR TITLE
296

### PR DESCRIPTION
**Pull request by jvmsangkal**

### CODE REVIEW

**Comment by jvmsangkal** [comment]
@nhvu1988 please check

File: analytics/index.html [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.96.1/js/materialize.min.js"></script>
+    <script type="text/javascript" src="../bower_components/react/react.js"></script>
+    <script src="../bower_components/react/JSXTransformer.js"></script>
+    <script type="text/javascript" src="../bower_components/toastr/toastr.min.js"></script>
+    <script type="text/javascript" src="../js/init.js"></script>
+    <script type="text/javascript" src="../js/config.js"></script>
+    <script type="text/javascript" src="../js/util.js"></script>
+    <script type="text/jsx" src="script.js"></script>
+</body>
+</html>
```
Comment (nhvu1988) [inline analytics/index.html]: new line at the end

File: analytics/index.html [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+<html>
+<head>
+    <title>Heartbeat analytics</title>
+    <link rel="icon" type="image/png" href="../images/favicon-hb-16x16.png">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/0.96.1/css/materialize.min.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="style.css">
+</head>
+<body>
+    <nav class='heartbeat_navbar red darken-2' role='navigation'>
```
Comment (nhvu1988) [inline analytics/index.html]: use double quote for html file ?

File: analytics/script.js [hunk 3]
```diff
@@ -0,0 +1,120 @@
+(function (htbt) {
+    'use strict';
+
+    var YTContainer =  React.createClass({
+        render: function () {
+            return (<div className='row'>
+                        <div className="col s12 m12">
```
Comment (nhvu1988) [inline analytics/script.js]: and use single quote in js file also?

File: analytics/script.js [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+                                    <div id="videos">
+                                    </div>
+                                </div>
+                            </div>
+                      </div>
+                </div>);
+        },
+        componentDidMount: function () {
+            // Set authorized scope.
+            var SCOPES = ['https://www.googleapis.com/auth/analytics.readonly'],
```
Comment (nhvu1988) [inline analytics/script.js]: why `scopes` is seem very special like that? :D

File: analytics/script.js [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+                authData = {
+                    client_id: htbt.config.gapi_client_id,
+                    scope: SCOPES,
+                    immediate: true
+                };
+
+            start();
```
Comment (nhvu1988) [inline analytics/script.js]: call start at the end of function is better, I think :)

File: analytics/script.js [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+            }
+
+            function render_result (result) {
+                _.forEach(result.rows, function (row) {
+                    $.get('https://www.googleapis.com/youtube/v3/videos?id=' +
+                        row[0] + '&key=' + htbt.config.yt_api_key +
+                        '&part=snippet,contentDetails,statistics,status')
+                    .done(function (data) {
+                        if (data.items.length) {
+                            $('#preloader').attr('style', 'display: none');
```
Comment (nhvu1988) [inline analytics/script.js]: can use `$().hide()` :)

File: analytics/script.js [hunk 7]
```diff
@@ -0,0 +1,120 @@
...
+                                    </div>
+                                </div>
+                            </div>
+                        </div>);
+            }
+    });
+
+    React.render(<YTContainer/>, document.getElementById('ytcontainer'));
+    React.render(<Loader/>, document.getElementById('videos'));
+})(window.htbt = window.htbt || {});
```
Comment (nhvu1988) [inline analytics/script.js]: new line at the end

Comment (jvmsangkal) [inline analytics/script.js]: fixed already.:)

**Pull request merged**